website/src/controllers/api-v3/groups.js

```diff
@@ -655,4 +656,58 @@ api.inviteToQuest = {
   },
 };
 
+/**
+ * @api {post} /groups/:groupId/quests/reject Reject a quest
+ * @apiVersion 3.0.0
+ * @apiName RejectQuest
+ * @apiGroup Group
+ *
+ * @apiParam {string} groupId The group _id (or 'party')
+ * @apiParam {string} questKey The quest _id
+ *
+ * @apiSuccess {Object} Quest Object
+ */
+api.rejectQuest = {
+  method: 'POST',
+  url: '/groups/:groupId/quests/reject/:questKey',
+  middlewares: [authWithHeaders(), cron],
+  async handler (req, res) {
+    let user = res.locals.user;
+    let questKey = req.params.questKey;
+    let quest = questScrolls[questKey];
+
+    req.checkParams('groupId', res.t('groupIdRequired')).notEmpty();
+
+    let validationErrors = req.validationErrors();
+    if (validationErrors) throw validationErrors;
+
+    let group = await Group.getGroup({user, groupId: req.params.groupId, fields: 'type quest'});
+    if (!group) throw new NotFound(res.t('groupNotFound'));
+    if (group.type !== 'party') throw new NotAuthorized(res.t('guildQuestsNotSupported'));
+    if (!quest) throw new NotFound(res.t('questNotFound', { key: questKey }));
+    if (!user.items.quests[questKey]) throw new NotAuthorized(res.t('questNotOwned'));
```

Comment:
These is the invitation object, I believe. So, the language may be off, but I am pretty sure this is checking to make sure the user has an invitation.